types/deneb/block_contents.yaml

```diff
@@ -1,37 +1,36 @@
 Deneb:
+  KZGProofs:
+    type: array
+    items:
+      $ref: '../primitive.yaml#/KZGProof'
+    minItems: 0
+    maxItems: 6
```

Comment:
Should use the theoretical max here, `MAX_BLOB_COMMITMENTS_PER_BLOCK`, for stable SSZ transport if the number changes in future

Reply:
Same for `Blobs`

Reply:
ref: https://github.com/ethereum/builder-specs/blob/534e4f81276b8346d785ed9aba12c4c74b927ec6/specs/deneb/builder.md#blindedblobsbundle

Reply:
+1 considering ssz transport

Reply:
good catch!

Reply:
updated!